scenarios/kubernetes_verify.py

```diff
@@ -27,7 +27,7 @@
 
 
 VERSION_TAG = {
-    '1.10': '1.10-v20181218-db74ab3f4',
+    '1.10': '1.10-v20180221-3655e24ae',
```

Comment:
Both versions have go1.9.3 installed...

Reply:
whoooo knows. this seems to fix something though /shrug

Reply:
either way we should be deleting this now-ish, just holding out for a possible last patch